src/controllers/authentication.controller.js

```diff
@@ -37,4 +37,16 @@ export default class AuthenticationControler {
       return errorResponse(res, statusCodes.badRequest, customMessages.userSignupFailed);
     }
   }
+
+  /**
+   * function findAll() returns all users in db
```

Comment:
does really this function find all users in the db?

Reply:
@mifeille I changed the description, it was an error I did